code/client/clrcore/External/Audio.cs

```diff
@@ -90,36 +90,42 @@ public static class Audio
 		};
 		#endregion
 
+		/// <remarks>This method allocates a sound ID. It is imperative you release the sound ID when you are done using it through <see cref="ReleaseSound"/></remarks>
 		public static int PlaySoundAt(Vector3 position, string sound, string set)
 		{
 			var soundId = API.GetSoundId();
 			API.PlaySoundFromCoord(soundId, sound, position.X, position.Y, position.Z, set, false, 0, false);
 			return soundId;
 		}
+        /// <inheritdoc cref="PlaySoundAt(Vector3, string, string)"/>
```

Comment:
Can you swap these with tab `\t` indentation like the rest of the file? While you're at it can you also add a new line between the methods as well? Will clean up things nicely.